keyboards/westberry/db190/rules.mk

```
@@ -0,0 +1,6 @@
+RGB_MATRIX_CUSTOM_USER = yes # Enable custom RGB effect
```

is at the keyboard level, should only enable kb version.
```suggestion
RGB_MATRIX_CUSTOM_KB = yes
```